src/operations/power.jl

```diff
@@ -222,7 +222,7 @@ Return the matrix represented by a wrapper of a matrix power.
 
 The matrix power represented by the wrapper.
 """
-function get(pow::IntervalMatrixPower)
+function materialize(pow::IntervalMatrixPower)
     return pow.Mᵏ
```

Comment:
The `get` reflects a bit more than it's a getter function, and `materialize` seems that is doing some computation, but it does't apparently.

`get` is bad, but how about `get_matrix` or just `matrix`?

Reply:
> If not, I would go with matrix.

sounds good

Reply:
done